tools/build_defs/repo/http.bzl

```diff
@@ -241,8 +241,9 @@ to omit the checksum as remote files can change._ At best omitting this
 field will make your build non-hermetic. It is optional to make development
 easier but either this attribute or `sha256` should be set before shipping.""",
     ),
-    "netrc": attr.string(
+    "netrc": attr.label(
```

Comment:
Let me know if this needs additional comments - or if there's additional documentation that needs updating.

Reply:
Maybe "A label that specifies the location of the .netrc file to use for authentication" is slightly better?

Reply:
Done.